### ISSUE TITLE
Support aliasing ?

### ISSUE THREAD

**Issue by doloopwhile** [mpppk/hlb#50]
Hello.

I am using `hlb` as an alternative of `hub`. However, it looks impossible to use `hlb` as an wrapper of `git`.

That is, once we define an alias...

```
# ~/.bashrs
alias git=hub
```

It is possible call git sub-commands and hub sub-commands with `git`.

```
$ git commit # git sub-comamnd
$ git browse # hub sub-command
```

Do you have a plan to support the aliasing?

Thanks.

**Comment by mpppk** [comment]
Yes, I'd like to make hlb commands available from git command.  
However, I'm considering to use mechanism of git subcommand instead of wrapping of git like `hub`.

For example, how about adding `install-as-git-subcommand` command that generate shell scripts for git subcommand.

```sh
$ hlb install-as-git-subcommand
# => generate shell scripts that named git-browse, git-create, etc...

$ cat git-browse
# /bin/bash
hlb browse $@

$ git browse
# => browser will be open!
```

This method has some problems compared to implementing `hlb` as git wrapper (for example, how we support Windows users...?), but there is no need to pay attention to other git subcommands for bypassing.

This is just one idea, I'm open to any suggestions or comments! :smile:
